full/build.gradle

```diff
@@ -48,7 +48,7 @@ dependencies {
 //    compile group: 'commons-codec', name: 'commons-codec', version: '1.14'
     compile group: 'com.jayway.jsonpath', name: 'json-path', version: '2.4.0'
     compile group: 'org.hdrhistogram', name: 'HdrHistogram', version: '2.1.9'
-    compile group: 'org.neo4j.driver', name: 'neo4j-java-driver', version: '4.0.0'
+    compile group: 'org.neo4j.driver', name: 'neo4j-java-driver', version: '4.2.0'
```

Comment:
Doing this we will deprecate the support to connect to Neo4j 3.4 which could be good for transferring data.
@mneedham wdyt?

Reply:
@conker84, according to [neo4j-java-driver Wiki](https://github.com/neo4j/neo4j-java-driver/wiki), `neo4j-java-driver 4.0` only supports server 3.5 and 4.0.
So according to official docs the current apoc 4.2.x.x release is neither compatible with server 3.4 nor with 4.1 or 4.2.

Reply:
I think this should be fine. I will merge it.